docs/modules/ROOT/pages/adr/0031-naming-scheme-for-servala-cluster-names-and-urls.adoc

```diff
@@ -0,0 +1,81 @@
+= ADR 0031 - Naming scheme for Servala cluster names and URLs
+:adr_author:    Nicolas Bigler
+:adr_owner:     Schedar
+:adr_reviewers: 
+:adr_date:      2025-04-11
+:adr_upd_date:  2025-04-11
+:adr_status:    draft
+:adr_tags:      servala,naming,scheme,cluster
+
+include::partial$adr-meta.adoc[]
+
+[NOTE]
+.Summary
+====
+With the launch of Servala we will have multiple clusters running on different cloud providers and on different regions. To ensure consistency and ease of management, it is important to establish a naming scheme for these clusters. This ADR outlines the proposed naming scheme for Servala cluster names and URLs.
+====
+
+== Context
+
+=== Cluster names
+
+We have two different kind of clusters for Servala:
+* Control-plane clusters: Those run Crossplane and contain all the Crossplane resources like functions, composites, api-server etc.
+* Service clusters: Those run the workloads that are managed by Crossplane.
+
+To properly distinguish and identify each cluster, the cluster name should contain the following information:
+* The CSP where the cluster is running: Cloudscale, Exoscale, etc.
+* The region where the cluster is running: `ch-gva-2`, `lpg1`, etc.
+* The type of cluster: `control` or `service`
+* The stage of the cluster: `test`, `prod`
+* A counter for each cluster to ensure uniqueness.
+
+Lieutenant already gives us part of the naming scheme. Each cluster should be prefixed with `c-` followed by the tenant, so that it can easily identify which tenant owns the cluster.
```

Comment:
Note that Lieutenant doesn't care whether you add the tenant to the cluster ID, since it only relies on the cluster's `spec.tenantRef.name` to establish the link to the tenant.

```suggestion
Lieutenant already gives us part of the naming scheme. Each cluster should be prefixed with `c-` followed by the tenant, so that it's easy for users to identify which tenant owns the cluster.
```